rsmtool/rsmsummarize.py

```diff
@@ -129,15 +139,29 @@ def run_summary(config_file_or_obj, output_dir):
         else:
             configpath = os.getcwd()
 
+    logger.info('Saving configuration file.')
+    configuration.save(output_dir)
```

Comment:
Same question as before. Were we just not doing this before?

Reply:
No :)